app.yaml

```diff
@@ -84,9 +84,9 @@ handlers:
 # under extensions in dev mode. This regex allows us to recursively serve js
```

Comment:
Minor note: you should modify this comment to add "custom ckeditor plugins".

Reply:
Done!